### DISCUSSION TITLE
How to show or hide more markers when zooming in and out?

### DISCUSSION THREAD

**Discussion by elie-chaaban** [react-native-maps/react-native-maps#4277] [Q&A]
## Why it is needed

If i have for example a track from a location to another location, and i m showing lots of markers between these two locations. How to do it in a way when the user zoom level is big, i show top rated places, when zooming in i show more in the zoomed region, etc..

I really need this, anyone implemented this?

**Comment by Willham12** [comment]
You can use clustering, see https://www.npmjs.com/package/supercluster
